ddtrace/context.py

```diff
@@ -25,8 +25,8 @@ class Context(object):
 
     This data structure is thread-safe.
     """
-    _partial_flush_enabled = asbool(get_env('tracer', 'partial_flush_enabled', default=False))
-    _partial_flush_min_spans = int(get_env('tracer', 'partial_flush_min_spans', default=500))
+    _partial_flush_enabled = asbool(get_env('tracer', 'partial_flush_enabled', default=True))
+    _partial_flush_min_spans = int(get_env('tracer', 'partial_flush_min_spans', default=5000))
```

Comment:
let's lower this, maybe 1k? lower _could_ work too. thinking about the scenarios that we typically have problems with, what is a lower bound of spans we'd consider as a "large" trace?